upgrade_analysis/models/upgrade_analysis.py

```diff
@@ -97,14 +126,34 @@ def analyze(self):
             }
         )
 
+        if not self.upgrade_path:
+            return (
+                "ERROR: no upgrade_path set when writing analysis of %s\n" % module_name
```

Comment:
Shouldn't this raise instead of returning?

Reply:
no, we don't want to have a blocking error here. if upgrade_path is not accessible, the analysis will be written in a data field (log). (same behaviour as before).